Themes/default/css/index.css

```diff
@@ -223,6 +225,8 @@ body#help_popup {
 }
 #likes li .like_time {
 	padding: 4px 0;
+	flex: 1 0 auto;
+	text-align: right;
```

Comment:
This seems like the float_right on the like_time should also be removed if this is a thing.

Reply:
We could do that, but leaving it in is harmless and requires one less change in the next patch file.